datadog-oci-orm/metrics-setup/connector_namespaces.py

```diff
@@ -0,0 +1,38 @@
+import json
+import sys
+
+MAX_NAMESPACES_PER_BATCH = 50
+MAX_COMPARTMENTS_PER_BATCH = 5
```

Comment:
I think parameterizing these constants probably can help us manage issues faster. If we take these as terraform variables, we should be able to adjust them without deploying a new stack to respond to potential issues that arise. What do you think ?

Reply:
Sounds good. One concern is these being low level details which can cause additional questions from someone running the stack and how these values influence the creation of connector hub.